### ISSUE TITLE
Previewing a status with spaces in it results in %20 in the rendered preview

### ISSUE THREAD

**Issue by scottmartinnet** [twitterdev/twitter-ruby-ads-sdk#89]
**Issue Type:** Bug

**Description:** When sending status text containing spaces to the API for a preview, the rendered preview contains %20s instead of spaces.  In https://github.com/twitterdev/twitter-ruby-ads-sdk/pull/67, I provided a code change to stop this, but learned that the bug was actually in the API.  @brandonblack reported that the bug has been fixed, but I'm still observing the same problem.

**Expected Result:**
"test test" is previewed as "test test"

**Actual Result:**
"test%20test" is previewed.

**Steps to Reproduce:**
- Send a string with spaces to the preview API.
- Observe the rendered preview.

**Relates To:** #67

**Comment by brandonblack** [comment]
Thanks for reporting @scottmartinnet. Let me check in again with the team that owns this service and confirm.

**Comment by hwz** [comment]
This API-related issue is now fixed

**Comment by brandonblack** [comment]
🎉🎉🎉

**Comment by scottmartinnet** [comment]
Fix confirmed from our end.

**Comment by scottmartinnet** [comment]
There appears to have been a regression here: the API is now rendering %20 again when this code is called with spaces.

**Comment by jbabichjapan** [comment]
@scottmartinnet Sorry to request you to do one more step - but do you mind posting this to https://twittercommunity.com/c/advertiser-api with repro steps so one of the team members can pick it up and look at it? (There are several people monitoring that forum for this sort of bug report)

**Comment by scottmartinnet** [comment]
Happy to, but I'm not 100% clear on whether this is a bug in the client or in the API itself.  The client code URI-escapes the input before sending it, which seems to clearly be the cause of this issue.  Removing that line stops the problem.  But the fix here was originally a change to the API, leaving the URI-escaping in place.  I had successfully tested the current client code (with URI-escaping) against the API after that fix.

If I filed a ticket with the API itself, it would say that sending URI-escaped text as a value renders that text as its URI-escaped value, which (naively) seems to be the right behavior.
